### PR TITLE
Retire inactive maintainers

### PR DESCRIPTION
**Pull request by ryjones**
Signed-off-by: Ry Jones <ry@linux.com>

#### Type of change

- Documentation update

#### Description

Removing maintainers who are not maintaining.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -12,11 +12,7 @@ See [the documentation on Maintainers](https://hyperledger-fabric.readthedocs.io
 | Artem Barger | [c0rwin][c0rwin] | c0rwin | <artem@bargr.net>
 | Senthilnathan Natarajan | [cendhu][cendhu] | Senthil1 | <cendhu@gmail.com>
 | Dave Enyeart | [denyeart][denyeart] | dave.enyeart | <enyeart@us.ibm.com>
-| Jay Guo | [guoger][guoger] | guoger | <guojiannan1101@gmail.com>
```
Comment (yacovm) [inline MAINTAINERS.md:15]: https://github.com/hyperledger/fabric/pull/3284

Comment (ryjones) [reply MAINTAINERS.md:15]: done

**Review by C0rWin — APPROVED** [review]

**Review by yacovm — APPROVED** [review]

**Review by andrew-coleman — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Review by manish-sethi — APPROVED** [review]

**Review by cendhu — APPROVED** [review]

**Comment by denyeart** [comment]
Majority approved, merging. Thanks for your many years of service Gari, Matt, Jason!

**Comment by mastersingh24** [comment]
@denyeart  - Why exactly am I inactive?  I do still try to review things, but frankly there was a quite a lull in meaningful activity which required review.  I'm pretty sure I had added comments to a few PRs and Issues in the past 90 days?

**Comment by denyeart** [comment]
@mastersingh24 I haven't seen a PR, review, meaningful engagement, or even attendance at a meeting this year, but if you become active again I would love to add you back.

**Pull request merged**